taipy/gui/page.py

```diff
@@ -180,3 +184,34 @@ def set_style(self, style: t.Dict[str, t.Dict[str, t.Any]]) -> Page:
 
     def _get_style(self):
         return self.__style
+
+    def script_paths(self, script_paths: t.Optional[t.Union[str, Path, t.List[t.Union[str, Path]]]]) -> Page:
```

Comment:
I though we agreed that the Page constructor would have an additional 'scripts' parameter (just like 'style').
What is this API?